scripts/components/dependencies_validator.ts

```diff
@@ -107,7 +107,15 @@ export class DependenciesValidator {
     const errors: Array<string> = [];
     for (const dependencyName of Object.keys(dependencyVersionsUsages)) {
       const dependencyVersionUsage = dependencyVersionsUsages[dependencyName];
-      if (dependencyVersionUsage.allVersions.size > 1) {
+
+      /**
+       * TODO: This is a temporary fix for execa version inconsistency. Remove this once execa version is fixed.
+       * Issue: https://github.com/aws-amplify/amplify-backend/issues/962
+       */
+      if (
+        dependencyVersionUsage.allVersions.size > 1 /* Issue-962 Start */ &&
+        dependencyName !== 'execa' /* Issue-962 End */
+      ) {
```

Comment:
as for this one. 

we should lift this as an option, so that it can be configured here https://github.com/aws-amplify/amplify-backend/blob/9f1a78bfce25f876a48ddbc83eca8f3f6d4c3657/scripts/check_dependencies.ts#L5-L18 .

Can be a follow up.

Reply:
https://github.com/aws-amplify/amplify-backend/issues/966